### DISCUSSION TITLE
Generating cy.js files from .feature files

### DISCUSSION THREAD

**Discussion by FrancescoMusmanno** [badeball/cypress-cucumber-preprocessor#1127] [General]
Hello,
I open this discussion regarding a feature that I don't know if it already exists or if it is possible to add. 
It would be great if given the .feature files it would be possible to run a command that would allow the cy.js files to be generated explicitly. 
For example, something like this:
```
describe('My First Test', () => {
  it('Visits the Kitchen Sink', () => {
    cy.visit('https://example.cypress.io')
  })
})
```
This way it would be possible to run tests using Cypress even in environments where it is not configured or not possible to configure this package. In this way it would be possible to increase the deliverability of tests and run them on environments where only Cypress is available. 

Thank you very much for your attention!

**Comment by badeball** [comment]
This package does a lot more than mere translation and requires a configured runtime to function properly.

> This way it would be possible to run tests using Cypress even in environments where it is not configured or not possible to configure this package.

I don't know what kind of use case this is, but it's out of scope.
